### ISSUE TITLE
Calc: Editing a formula does not highlight referenced cells

### ISSUE THREAD

**Issue by laurentS** [CollaboraOnline/online#9195]
### Describe the Bug
When double clicking a cell with a formula (say a simple sum), I would expect referenced cells to be highlighted for easier understanding of the formula.

### Steps to Reproduce
1. Fill in 2-3 cells with numbers
2. In another cell, create a formula referencing them, such as `=cell1+cell2+cell3`
3. Edit some other cell
4. Double click the formula cell to edit its formula

### Expected Behavior
Referenced cells from step 1 should be highlighted

### Actual Behavior
They are not. The formula itself shows each referenced cell name in a different colour, but the cells themselves are not.

### Screenshots
This is Collabora version: 24.04.2.1:
![image](https://github.com/CollaboraOnline/online/assets/1745184/81400ddf-3d6b-4300-9957-24224e830475)

Meanwhile in LibreOffice 24.2.3.2:
![image](https://github.com/CollaboraOnline/online/assets/1745184/df21c8a6-1929-4d15-89f1-e088bbbc8a50)

### Desktop
(Please complete the following information)
 - **Collabora version:** 24.04.2.1
 - **OS and version:** Debian/linux sid
 - **Browser and version:** Firefox 126.0

### Smartphone
Not tested

**Comment by thebearon** [comment]
Indeed, I can confirm, and it'is a regression from the following commit:
https://github.com/CollaboraOnline/online/commit/58e5208027ee58a1ac6c071eb81ac78db4183649
https://github.com/CollaboraOnline/online/pull/8963
```
Author:     Szymon Kłos <szymon.klos@collabora.com>
AuthorDate: Fri May 3 12:44:51 2024 +0200
Commit:     Szymon Kłos <eszkadev@gmail.com>
CommitDate: Sun May 5 21:22:55 2024 +0200

    Fix buttons visibility in formulabar on cancelformula
```

Thanks for reporting!

**Comment by thebearon** [comment]
Should be fixed by #9207.
